README.md

```diff
@@ -28,13 +28,13 @@ module.exports = NextOidc({
   clientSecret: 'P8WDmH4XyumzkTCx...',
   scope: 'openid profile',
   domain: 'sandrino-nextjs.auth0.com',
-  redirectUri: 'http://localhost:3000/callback',
-  postLogooutRedirectUri: 'http://localhost:3000/',
+  redirectUri: 'http://localhost:3000/api/callback',
```

Comment:
It seems that the example domain doesn't like this `redirectUri`, probably needs to be updated on that end also

<img width="1441" alt="Screen Shot 2019-08-31 at 11 49 35 AM" src="https://user-images.githubusercontent.com/22380829/64066922-86bd1a00-cbe5-11e9-8adf-de88ce6393d7.png">

Reply:
Yes, I tested it using `/callback`, but I did the push with `/api/callback`. The url has to be changed in the Auth0 Admin too (In @sandrinodimattia account). I'm adding a note for it, thank you.